tests/devices/test_default_gaussian.py

```diff
@@ -373,7 +373,8 @@ def test_operation_map(self, gaussian_dev):
             "Kerr",
         }
 
-        assert set(qml.ops._cv__ops__) - non_supported == set(gaussian_dev._operation_map)
+        cv_ops = set(qml.ops._cv__ops__) | {"Identity"}  # we also support identity operation
```

Comment:
It would be great to have a test case in `TestGates` too that checks applying the identity.

Reply:
Got it !